scio-examples/src/main/java/org/apache/beam/examples/common/ExampleCloudSqlOptions.java

```diff
@@ -0,0 +1,46 @@
+package org.apache.beam.examples.common;
+
+import org.apache.beam.sdk.options.Default;
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.options.PipelineOptions;
+import org.apache.beam.sdk.options.Validation;
+
+public interface ExampleCloudSqlOptions extends PipelineOptions {
```

Comment:
Users may have different ways of providing all the information that need to initiate the database connection(i.e config file as labianchin mentioned in his comments) , I don't think enforcing users to provide it through command line everytime is good.